src/science/mathematics/linear-algebra/vector_sum.md

```diff
@@ -0,0 +1,49 @@
+## Vector Sum
+[![ndarray-badge]][ndarray]
+
+The [ndarray] crate supports a number of ways to create arrays -- in this
+example we focus on creating arrays from `std::Vec` via [`from_vec`]. Adding two
+arrays together is no different than adding two numbers together. Using the `&`
+operand on the arrays within an arithmetic operation prevents the operation from
+consuming the arrays. Without `&`, the arrays are consumed.
+
+In the first example, arrays `a` and `b` are moved in the operation that creates
+array `z`. In the second example, the arrays `c` and `d` are not moved and
+instead, a new array is created for `w`. Updating either of `c` or `d` after the
+vector sum has no effect the value of `w`.
+
+```rust
+extern crate ndarray;
+use ndarray::Array;
```

Comment:
can we get an API link to this in the description?

Reply:
I use the name `ndarray::Array` but I actually link to `ndarray::ArrayBase`. We are creating `ndarray::Array`s, but the useful API documentation is `ndarray::ArrayBase`. I was undecided whether it was worth explicitly calling this out but decided against it as it may cause too much confusion.